compiler/test/compilable/ctod.i

```diff
@@ -7,7 +7,16 @@ TEST_OUTPUT:
 ---
 === ${RESULTS_DIR}/compilable/ctod.di
 // D import file generated from 'compilable/ctod.i'
-extern (C) uint equ(double x, double y);
+extern (C)
+{
+	uint equ(double x, double y);
+	enum SQLINTERVAL
```

Comment:
This should be anonymous - otherwise access to members would be different between C and Di.